DOCUMENTATION.md

```diff
@@ -1672,6 +1673,57 @@ This class allows player movement based on grabbing of `VRTK_InteractableObject`
 
 ---
 
+## Dash Teleporter (VRTK_DashTeleport)
+
+### Overview
+
+The dash teleporter extends the height adjust teleporter and allows to have the `[CameraRig]` dashing to a new teleport location. 
+
+Like the basic teleporter the Dash Teleport script is attached to the `[CameraRig]` prefab and requires a World Pointer to be available.
+
+The basic principle is to dash for a very short amount of time, to avoid sim sickness. The default value is 100 miliseconds. This value is fixed for all 
+normal and longer distances. When the distances get very short the minimum speed is clamped to 50 mps, so the dash time becomes even shorter.
+This idea is taken from Realities' "ludicrous speed".
+The minimum distance for the fixed time dash is determined by the minSpeed and normalLerpTime values, if you want to always lerp with a fixed mps speed instead, set the normalLerpTime to a high value. 
+Right before the teleport a capsule is cast towards the target and registers all colliders blocking the way. These obstacles are then broadcast in an event so that for example their gameobjects or renderers can be turned off while the dash is in progress.
+
+### Inspector Parameters
+
+  * **Blink Transition Speed:** The fade blink speed on teleport. Probably should best be set to a value of 0 for dashing.
```

Comment:
Make sure you get the most up to date params from the height adjust teleport docs as looks like you're missing a couple (plus the limit to nav mesh no longer exists)